socorro/processor/mozilla_transform_rules.py

```diff
@@ -851,7 +849,14 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
             # We apply this Rule only if the release channel is beta, because
             # beta versions are the only ones sending an "incorrect" version
             # number in their data.
-            return processed_crash['release_channel'].lower() == 'beta'
+            # 2017-06-14: Ohai! This is not true anymore! With the removal of
+            # the aurora channel, there is now a new type of build called
+            # "DevEdition", that is released on both the beta and the aurora
```

Comment:
For me that comment only says that there are versions on the `beta` & `aurora` update channels that share the same versions. Which is because Firefox Beta uses the `beta` update channel and Firefox Developer Edition uses the `aurora` update channel which is now more or less equal to the beta one.